crates/core/src/fnc/mod.rs

```diff
@@ -1187,6 +1188,7 @@ pub async fn idiom(
 				"is_ascii" => string::is::ascii,
 				"is_datetime" => string::is::datetime,
 				"is_domain" => string::is::domain,
+				"is_edge" => record::is::edge((stk, ctx, Some(opt), doc)).await,
```

Comment:
Not sure if we should add this on a string. If you want to do such a check you should just convert the value to a record first...